indexer/services/ender/src/handlers/order-fills/abstract-order-fill-handler.ts

```diff
@@ -289,6 +290,10 @@ export abstract class AbstractOrderFillHandler<T> extends Handler<T> {
     return updatedPerpetualPosition;
   }
 
+  /**
+   * @param isCanceled - if the order is in the CanceledOrderCache, always false for liquidiation
+   * orders
+   */
```

Comment:
nit: Create a full description here for the function. Confusing to only describe a single parameter.